src/modules/dashboard/components/ColonyGrid/ColonyGridItem.tsx

```diff
@@ -1,54 +1,46 @@
 import React from 'react';
 
-import { useDataFetcher, useDataSubscriber } from '~utils/hooks';
 import Heading from '~core/Heading';
 import Link from '~core/Link';
 import { SpinnerLoader } from '~core/Preloaders';
 import HookedColonyAvatar from '~dashboard/HookedColonyAvatar';
-import { log } from '~utils/debug';
 import { ColonyProps } from '~immutable/index';
+import { useColonyQuery } from '~data/index';
 
-import { colonyNameFetcher } from '../../fetchers';
 import styles from './ColonyGridItem.css';
 
-import { colonySubscriber } from '../../subscribers';
-
 type Props = ColonyProps<'colonyAddress'>;
 
 const ColonyAvatar = HookedColonyAvatar({ fetchColony: false });
 
 const ColonyGridItem = ({ colonyAddress }: Props) => {
-  const { isFetching, data: colony } = useDataSubscriber(
-    colonySubscriber,
-    [colonyAddress],
-    [colonyAddress],
-  );
-  const { data: colonyName, isFetching: isFetchingColonyName } = useDataFetcher(
-    colonyNameFetcher,
-    [colonyAddress],
-    [colonyAddress],
-  );
+  const { data, loading } = useColonyQuery({
```

Comment:
Shouldn't this use the `colonyFetcher` asa well ?

Reply:
It's not always clear, that I am sorry for. But here we just need the information that's in the db